sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -56,6 +57,14 @@ internal class RadarActivityLifecycleCallbacks : Application.ActivityLifecycleCa
         foreground = count > 0
 
         updatePermissionsDenied(activity)
+
+        val appOpenMetadata = JSONObject().put("amount", 1)
+        Radar.logConversion(
+            "app_open",
+            appOpenMetadata
+        ) { status, location, events, user ->
+            Log.i(TAG, "Custom event type = ${events?.first()?.customType}: status = $status; location = $location; events = $events; user = $user")
```

Comment:
why do we log `TAG` here?

Reply:
not sure why. the other logs in this class use it, but it's probably not useful here and can be set to null